mollymawk fails to build on builder-web

```
411.929s:# File "unikernel.ml", line 3165, characters 64-78:
411.929s:# 3165 |                          (unikernel_prepare_update stack store ~unikernel_name
411.929s:#                                                                        ^^^^^^^^^^^^^^
411.929s:# Error: The function applied to this argument has type
411.929s:#          Dune__exe.Albatross.t ->
411.929s:#          Vmm_core.Name.Label.t ->
411.929s:#          'a -> Dune__exe.User_model.user -> H1.Reqd.t -> unit Lwt.t
411.929s:# This argument cannot be applied with label ~unikernel_name
411.929s:# gmake: *** [Makefile:46: build] Error 1
```

--- a/unikernel.ml
+++ b/unikernel.ml
@@ -3158,15 +3158,11 @@ struct
                         http_client)))
         | "/unikernel/update/compare-changes" ->
             check_meth `GET (fun () ->
-                match get_query_parameter "unikernel" with
-                | Ok unikernel_name ->
-                    authenticate store reqd
-                      (albatross_instance req.H1.Request.target
-                         (unikernel_prepare_update stack store ~unikernel_name
-                            http_client))
-                | Error err ->
-                    Middleware.http_response ~api_meth:false ~data:(`String err)
-                      reqd `Bad_request)
+                authenticate store reqd
+                  (albatross_instance req.H1.Request.target (fun albatross ->
+                       unikernel
+                         (unikernel_prepare_update stack store http_client
+                            albatross))))
         | _ ->
             Middleware.http_response ~api_meth:false ~title:"Page not found"
               ~data:(`String "This page cannot be found.") reqd `Bad_request)

